react-backend/client/src/components/authentication/signupcb.js

```diff
@@ -58,10 +56,16 @@ export default class CBsignup extends Component {
         <h1>Please provide us with required information on your business</h1>
         <form className="Signup" onChange={this.handleChange} onSubmit={this.handleSubmit}>
           <Input question="Business Name" option="org_name" />
-          {errors.orgName && <div className="ErrorText">Business name: {errors.orgName.map(e => <span>{e}</span>)}</div>}
+          {errors.orgName && (
```

Comment:
Did anything change in this file or is it just prettier layout?

Reply:
Just prettier, I had like 3 merge conflicts on this file when I rebased, so the formatting is probably different